src/test/java/seedu/address/model/person/ExperienceTest.java

```diff
@@ -27,15 +27,15 @@ public void isValidExperience() {
         // invalid experience
         assertFalse(Experience.isValidExperience("")); // empty string
         assertFalse(Experience.isValidExperience(" ")); // spaces only
-        assertFalse(Experience.isValidExperience("^")); // only non-alphanumeric characters
+        assertFalse(Experience.isValidExperience("^!@#$%")); // only non-alphanumeric characters
         assertFalse(Experience.isValidExperience("11*")); // contains non-alphanumeric characters
         assertFalse(Experience.isValidExperience("1.11")); // contains decimals
         assertFalse(Experience.isValidExperience("-11")); // contains negative number
-
-
+        assertFalse(Experience.isValidExperience("12345")); // number is too large
 
         // valid experience
         assertTrue(Experience.isValidExperience("1")); // positive number
-        assertTrue(Experience.isValidExperience("12345")); // numbers only
+        assertTrue(Experience.isValidExperience("67")); // number not larger than 67
```

Comment:
Instead of using the number `67`, I think you should use `Experience.MAX_EXPERIENCE`. So that if this number has to be changed in the future, you won't have to individually find every instance of `67`. Additionally, it improves code quality (magic literals)

Reply:
Agree!